### PR TITLE
add edit function

### PR DESCRIPTION
**Pull request by rumaf**

### CODE REVIEW

**Comment by rumaf** [comment]
This the best commit ever!!!!

File: app/actions.rb [hunk 1]
```diff
@@ -12,6 +12,13 @@
 # Routes for edit
 
 get '/:id/edit' do
+	@videos = Video.find(params[:id])
+	erb :edit
+end
+
+post '/:id/edit' do
+	@videos = Video.where(id: params[:id]).update_all(title: params[:title], year: params[:year], director: params[:director])
```
Comment (patriciamadureira87) [inline app/actions.rb:20]: Please correct this indentation!

**Comment by patriciamadureira87** [comment]
recheck your code!!! improve your indentation!!

**Comment by rumaf** [comment]
What empty line? Please be more specific! :-1:

**Pull request merged**